kuma/urls.py

```diff
@@ -100,6 +101,7 @@
         name="redirect-to-payments",
     ),
 )
+urlpatterns += i18n_patterns(re_path(r"^accounts/", include(accounts_lang_urlpatterns)))
```

Comment:
I don't understand this. 

* On line 48 of route.jsx it says `/${this.locale}/account` and here's it's `accounts/`
* In my browser address bar I don't want to see "accounts" because that sounds to me like that's where I manage my different accounts. Like a list of my different providers. A place where it lists my different google and github logins. 
* What is already there? "account clashes with an existing route we have". 

One last time, are we sure "settings" isn't better? I think it's rather important to nail this. A word that works for the basic stuff like setting your name, managing your subscription, administer your browser sets, etc. What other future things are going to go there and do they sound better with "settings" or "account"?

Reply:
> On line 48 of route.jsx it says `/${this.locale}/account` and here's it's `accounts/`

My bad, it should be `accounts`

> In my browser address bar I don't want to see "accounts" because that sounds to me like that's where I manage my different accounts. Like a list of my different providers. A place where it lists my different google and github logins.

I completely agree

> What is already there? "account clashes with an existing route we have".

I went off an error thrown by `web` in the logs but, I think it might be this here: https://github.com/mdn/kuma/blob/53780fa48f38456ffaa4a5e234426a05eddf0ba0/kuma/users/urls.py#L57

> One last time, are we sure "settings" isn't better? I think it's rather important to nail this. A word that works for the basic stuff like setting your name, managing your subscription, administer your browser sets, etc. What other future things are going to go there and do they sound better with "settings" or "account"?

😬 I think `account` would work but, the more I think about it, and seeing we already ran into a problem, I have been convinced to switch to using `settings` 🎉

Reply:
Ah :hankey: we already have `settings` as well :laughing:

Reply:
`preferences` worked out 🍭